### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DanielMitiku**

### CODE REVIEW

File: css/framework.css [hunk 1]
```diff
@@ -0,0 +1,226 @@
+/* reset */
```
Comment (oluwoleilesanmi) [inline css/framework.css]: Extract section reset into a reset.css file

Comment (DanielMitiku) [reply css/framework.css]: just did that

File: css/framework.css [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+.fixed-nav{
+    clear: both;
+    position: fixed;
+    width: 100%;
+    height: auto;
+    background-color: black;
+    display: flex;
+
+}
+.fixed-nav ul{
```
Comment (oluwoleilesanmi) [inline css/framework.css]: Avoid `.fixed-nav ul` in the framework class.
Instead `.fixed-nav` in the framework class dont call elements directly, just create selectors.
- The above is good separation of a concern.

**Review by oluwoleilesanmi — CHANGES_REQUESTED** [review]
@DanielMitiku @IBTechRaj nice work, you are almost there make the changes below and you will be good to go.
- In framework.css avoid calling any element in html directly, only use framework.css to create selectors
- Navigate using below to see an example.
```
https://960.gs/demo.html >> Inspect element (developer tools) >> Sources >> Css
```

**Review by oluwoleilesanmi — APPROVED** [review]
@IBTechRaj @DanielMitiku well done i really like your work really clean. :1st_place_medal: :+1: Proceed.

**Pull request merged**